packages/terra-application-utility/tests/jest/UtilityMenu.test.jsx

```diff
@@ -100,4 +100,42 @@ describe('UtilityMenu', () => {
     const wrapper = shallow(component);
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should render with a user specified role', () => {
+    const role = 'menu';
+    const component = (
+      <IntlProvider locale={locale} messages={messages}>
+        <HeaderUtilityMenu
+          initialSelectedKey="key"
+          menuItems={config}
+          onChange={mockOnChange}
+          onDisclose={mockOnDisclose}
+          variant={variant}
+          role={role}
```

Comment:
```suggestion
          menuRole={role}
```

Reply:
[Changed to menurole](https://github.com/cerner/terra-framework/pull/501/commits/131aaf225ec1730dafcfecf63f4d79d0857c6bfa)

Reply:
update to camelCase - `menuRole`